app/webpacker/components/Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt/ActionItems.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { List } from 'semantic-ui-react';
+import { competitionUrl } from '../../../../lib/requests/routes.js.erb';
+
+export default function ActionItems({ actionItemList, messageArgs }) {
+  return (
+    <List bulleted>
+      {actionItemList.map((actionItem) => (
+        <List.Item>
+          <ActionItemContent
+            actionItem={actionItem}
+            messageArgs={messageArgs}
+          />
```

Comment:
I think you would be much better off letting I18n handle this. It would definitely save you from the `messageArgs` with the tons of `?` safe operations, and it would also allow you to unify `ActionItems` and `NonActionItems` like so:
```yml
en:
  anonymization:
    user_currently_banned:
      true: This user is currently banned and cannot be anonymized.
      false: This user is currently not banned.
```